app/js/providers/stock.js

```diff
@@ -1,5 +1,7 @@
 'use strict';
```

Comment:
We should use fetch cause I already require it some where and it is better and all but  I mean we don't have to